frankly034/Dantown
# #168493530 setup email verification

#### What does this PR do?
Setup email verification feature 

#### Description of Task to be completed?
- install mailgun and sendgrid
- setup mailgun email templates
- add account verification controller and routes

#### How should this be manually tested?
* clone the repo
* run `npm install`
* start the app with `npm run start:dev`
* signup by visiting `POST` `localhost:3000/api/v1/auth/signup`
* check your email and click on the verification button
* You should get a `200` status on success or `400` on failure

#### Any background context you want to provide?
* must have `node` installed

#### What are the relevant pivotal tracker stories?
[Pivotal tracker story #168493530](https://www.pivotaltracker.com/n/projects/2385884/stories/168493530)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
